deployments/test/4_authorization.spec.ts

```diff
@@ -206,7 +206,7 @@ describe('Deployment: Authorization', () => {
 
     it('core contains kyber network exchange wrapper', async () => {
       const kyberNetworkAddress = await getContractAddress(KyberNetworkWrapper.contractName);
-      expect(kyberNetworkAddress).toContain(kyberNetworkAddress);
+      expect(exchanges).toContain(kyberNetworkAddress);
```

Comment:
Side note, do we plan to even deploy the 0x exchange wrapper with our upcoming deploy? I thought we were punting